operator/v1/types_ingress.go

```diff
@@ -258,6 +258,74 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	HTTPCompression HTTPCompressionPolicy `json:"httpCompression,omitempty"`
+
+	// IdleConnectionTerminationPolicy maps directly to HAProxy's
```

Comment:
Nit, godoc should start with the serialised version of the string. We do this so that `oc explain` looks correct

Reply:
Fixed in https://github.com/openshift/api/compare/1c1e45f0178b3f54852437b752964e92f019bed1..000e4bc7faace3e166e649c8bba32db0b1570c13